modules/swagger-codegen/src/main/resources/r/model.mustache

```diff
@@ -76,6 +76,22 @@
         {{/isListContainer}}
         {{/vars}}
       )
+    },
+    fromJSON = function({{classname}}Json) {
+      {{classname}}Object <- jsonlite::fromJSON({{classname}}Json)
+      {{#vars}}
+      {{#isPrimitiveType}}
+      self${{name}} <- {{classname}}Object${{name}}
+      {{/isPrimitiveType}}
+      {{^isPrimitiveType}}
+      {{#isListContainer}}
+      self${{name}} <- lapply({{classname}}Object${{name}}, function(x) {{datatype}}$new()$fromJSON(x)
+      {{/isListContainer}}
+      {{^isListContainer}}
+      self${{name}} <- {{datatype}}$new()$fromJSON({{classname}}Object${{name}})
```

Comment:
self${{name}} <- {{datatype}}$new()$fromJSON(jsonlite::toJSON({{classname}}Object${{name}}))

Reply:
I will also change line 88 from 
```
      self${{name}} <- lapply({{classname}}Object${{name}}, function(x) {{datatype}}$new()$fromJSON(x)
```
to 
```
      self${{name}} <- lapply({{classname}}Object${{name}}, function(x) {{datatype}}$new()$fromJSON(jsonlite::toJSON(x))
```